ThunderCloud/MultiVideoPlayerViewController.swift

```diff
@@ -140,9 +157,30 @@ open class MultiVideoPlayerViewController: UIViewController {
 			player = nil
 			videoPlayerLayer?.removeFromSuperlayer()
 		}
-		
-		UIDevice.current.setValue(Int(UIInterfaceOrientation.portrait.rawValue), forKey: "orientation")
+        
+        // If we can't get the supported orientations rotate away as we don't want to make any assumptions
+        guard let supportedOrientationStrings = Bundle.main.infoDictionary?["UISupportedInterfaceOrientations"] as? [String] else {
+            rotateDeviceToPortrait()
+            return
+        }
+        
+        // If we don't get the same number of enum's as we do the original strings we don't want to assume anything about orientationss
+        let supportedOrientations = supportedOrientationStrings.flatMap({ UIInterfaceOrientation(stringValue: $0) })
+        guard supportedOrientations.count == supportedOrientationStrings.count else {
+            rotateDeviceToPortrait()
+            return
+        }
+		
+        guard !supportedOrientations.contains(UIApplication.shared.statusBarOrientation) else {
+            return
+        }
+        
+		rotateDeviceToPortrait()
 	}
+    
+    private func rotateDeviceToPortrait() {
+        
```

Comment:
So... this method is empty?

Reply:
@MattCheetham Mah bad, don't know what's wrong with me! Sorted